proxy/src/registry.rs

```diff
@@ -551,6 +584,29 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    async fn test_list_org() {
+        // Test that a registered org can be retrieved.
+        let client = Client::new_emulator();
+        let mut registry = Registry::new(client.clone());
+        let alice = ed25519::Pair::from_legacy_string("//Alice", None);
+
+        // Register the org
+        let org_id = OrgId::try_from("monadic").unwrap();
+        let registration = registry
+            .register_org(&alice, org_id.clone().into(), 10)
+            .await;
+        assert!(registration.is_ok());
+
+        // List the orgs
+        let orgs = registry
+            .list_orgs(&ed25519::Public::from(alice.clone()))
```

Comment:
Can't we just call `public` on `alice` to get the pub key?

Reply:
b0d295c33b8b3f4a147febaa3eae77c977765302